Conductor/Client/Configuration.cs

```diff
@@ -119,35 +119,19 @@ public Configuration()
         /// </summary>
         public Configuration(
             IDictionary<string, string> defaultHeader,
-            IDictionary<string, string> apiKey,
```

Comment:
We still need to support apiKey and apiSecret otherwise it will break for the existing sdk users. We can initialise OrkesApiClient inside Configuration but we should not remove/change the configuration signature.

Reply:
You removed it, I think you can add it again then.

Reply:
Its ok. I am working on cleaning up few things and supporting backward compatibility.  We can merge this PR.